Podfile

```diff
@@ -16,6 +16,10 @@ def shared_pods
   #pod 'GRKOpenSSLFramework', path: '../GRKOpenSSLFramework'
 
   # third party pods
+  pod 'AFNetworking', inhibit_warnings: true
+  pod 'JSQMessagesViewController',  git: 'https://github.com/WhisperSystems/JSQMessagesViewController.git', branch: 'mkirk/share-compatible', :inhibit_warnings => true
```

Comment:
Avatar building uses JSQ classes, but JSQ is not framework compatible. I think removing this dependency will be straight forward, but I left it as a TODO to get this PR up.

In the meanwhile, I just hacked JSQ to comment out the corresponding methods.

Reply:
Okay, I would _love_ to remove our dependency on JSQ.  I've got a bunch of work in that direction in the "message bubble" branch.

Reply:
Made a card.